yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -352,9 +352,9 @@ func buildCallhomeSchemaOptimizationChanges() []callhome.SchemaOptimizationChang
 			Objects:          getAnonymizedIndexObjectsFromIndexToTableMap(schemaOptimizationReport.RedundantIndexChange.TableToRemovedIndexesMap),
 		})
 	}
-	if schemaOptimizationReport.TableShardingRecommendation != nil {
+	if schemaOptimizationReport.TableColocationRecommendation != nil {
 		objects := make([]string, 0)
-		for _, obj := range schemaOptimizationReport.TableShardingRecommendation.ShardedObjects {
+		for _, obj := range schemaOptimizationReport.TableColocationRecommendation.ShardedObjects {
```

Comment:
Ideally, we should not need/avoid to modify our struct as it can have a cascading effect.

In this case, i think we are not using the struct apart from passing to html template?

Reply:
right, there can't any breaking changes issue with as this struct right now is only relevant for export schema command whcih is not resumable

Reply:
Yes